### ISSUE TITLE
Doppelpunkte escapen

### ISSUE THREAD

**Issue by saerdnaer** [muccc/wiki_kalenderscraper#6]
Aktuell gibt es auf der Kalender-Wikiseite einen Eintrag ``Save the date: µC³ Party``. Bei mir im Kalender wird daraus nur noch "Save the date": 
<img width="278" alt="" src="https://cloud.githubusercontent.com/assets/40266/19625200/54c7da3e-9912-11e6-8915-ed086951d8bc.png">

In der ics Datei steht es wie folgt:
```
BEGIN:VEVENT
SUMMARY:Save the date
DTSTART;VALUE=DATE-TIME:20161029T172300
DTEND;VALUE=DATE-TIME:20161029T192300
UID:wikical29.10.Savethedate.17.23@api.muc.ccc.de
DESCRIPTION:Public
LOCATION:Alle
END:VEVENT
```

**Comment by derchrisuk** [comment]
Hast du eine alte ICS Datei?

Sieht bei mir so aus:

`BEGIN:VEVENT
SUMMARY:µC³ Party
DTSTART;VALUE=DATE-TIME:20161029T172300
DTEND;VALUE=DATE-TIME:20161029T192300
UID:wikical29.10.µC³Party.17.23@api.muc.ccc.de
DESCRIPTION:Public
LOCATION:Alle
END:VEVENT`

**Comment by robinkaranu** [comment]
Ja, nachdem das Datum ja inzwischen ist, habe ich aus der Kalenderquelle
das Prefix 'save the date: ' entfernt. :-)

-- krobin

**Comment by derchrisuk** [comment]
Denke aber das das ein generelles Problem ist, schau ich mir die Tage mal an
